docker-micro-benchmark/helpers/errors.go

```diff
@@ -0,0 +1,82 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package helpers
+
+import (
+	"fmt"
+	"sync"
+)
+
+const errorTopNum = 5
+
+// Error is the error returned by functions in docker_helpers
+type Error struct {
```

Comment:
nit: since you record more than just errors (e.g., you also record the total operations), maybe you can use a more general name (e.g., `Stats`?)